include/swift/Driver/Compilation.h

```diff
@@ -234,7 +238,8 @@ class Compilation {
               Optional<unsigned> BatchSizeLimit = None,
               bool SaveTemps = false,
               bool ShowDriverTimeCompilation = false,
-              std::unique_ptr<UnifiedStatsReporter> Stats = nullptr);
+              std::unique_ptr<UnifiedStatsReporter> Stats = nullptr,
+              bool EnableExperimentalDependencies = false);
```

Comment:
Getting convinced we need a CompilationOptions struct, but that doesn't have to be part of this patch.

Reply:
Yes, good points.